homeassistant/components/zwave_me/number.py

```diff
@@ -40,7 +40,7 @@ class ZWaveMeNumber(ZWaveMeEntity, NumberEntity):
     """Representation of a ZWaveMe Multilevel Switch."""
 
     @property
-    def native_value(self):
+    def native_value(self) -> float:
```

Comment:
Or maybe it is always an int?
```suggestion
    def native_value(self) -> int:
```